packages/@pollyjs/adapter-node-http/src/-private/transport-wrapper.js

```diff
@@ -270,13 +267,17 @@ export default class TransportWrapper {
         parsedUrl.set('hostname', hostname);
         parsedUrl.set('port', port !== '80' ? port : '');
 
-        adapter.handleRequest({
-          method,
-          headers,
-          url: parsedUrl.href,
-          body: chunks,
-          requestArguments: [wrapper, req, args]
-        });
+        adapter
+          .handleRequest({
+            method,
+            headers,
+            url: parsedUrl.href,
+            body: chunks,
+            requestArguments: [wrapper, req, args]
+          })
+          .catch(e => {
+            req.emit('error', e);
```

Comment:
Can you add you commit message as an inline message so future readers will understand why this is necessary? 

```
This allows the consumer to handle the error gracefully (there was no way to detect this error previously)
```